src/operator/random/sample_op.cc

```diff
@@ -171,5 +172,23 @@ Example::
 .set_attr<FCompute>("FCompute<cpu>", Sample_<cpu, GeneralizedNegativeBinomialSampler<cpu>>)
 .set_attr<FComputeEx>("FComputeEx<cpu>", SampleEx_<cpu, GeneralizedNegativeBinomialSampler<cpu>>);
 
+MXNET_OPERATOR_REGISTER_SAMPLE(_random_randint, SampleRandIntParam)
+.add_alias("random_randint")
+.describe(R"code(Draw random samples from a uniform distribution.
+
+Samples are uniformly distributed over the half-open interval *[low, high)*
```

Comment:
It needs to be made clear that this sampler is sampling from a discrete uniform distribution and that the sampled values are all integers.

Reply:
Yup, noted